### PR TITLE
Adds a fake sun object for mapper usage

### PR DESCRIPTION
**Pull request by Arokha**
If you put one of these on your map, it'll pick a random color/brightness from your choices, to pretend the map has a different time of day each time it's loaded. JUST PUT ONE THOUGH OKAY??? You can change that possible_light_setups to whatever you'd like for your options! Hooray.

Also randomly fixes ice thinking it has edges but it doesn't.

### CODE REVIEW

**Comment by Very-Soft** [comment]
YAY! <3

File: vorestation.dme [hunk 1]
```diff
@@ -2425,6 +2425,7 @@
 #include "code\modules\lighting\lighting_area.dm"
 #include "code\modules\lighting\lighting_atom.dm"
 #include "code\modules\lighting\lighting_corner.dm"
+#include "code\modules\lighting\lighting_fake_sun.dm"
```
Comment (Novacat) [inline vorestation.dme:2428]: This should probably have _vr appended to the end so I know it is a Vorestation specific file.

**Review by Novacat — CHANGES_REQUESTED** [review]

File: vorestation.dme (hunk 1 above)
Comment (Arokha) [reply vorestation.dme]: So piiiiicky

**Review by Novacat — APPROVED** [review]
The snow thing should probably be ported to Polaris but that is very smol

**Pull request merged**